### PR TITLE
Add Discord.js example

### PR DESCRIPTION
**Pull request by bddylol**
Example, Taken from my other PR and taken you're notes and adding it!

### CODE REVIEW

File: example.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Require the necessary discord.js classes
+const { Client, Intents } = require('discord.js');
+const { token } = require('./config.json');
+const roverify = require("roverify");
+
+// Create a new client instance
+const client = new Client({ intents: 513 });
```
Comment (MoaufmKlo) [inline example.js]: Using other intents here may be a good idea; as the [guide](https://discordjs.guide/popular-topics/intents.html) shows, you can use strings here instead.

File: example.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+// Intents Calc: https://ziad87.net/intents/
+
+// When the client is ready, run this code (only once)
+client.once('ready', () => {
```
Comment (MoaufmKlo) [inline example.js]: Please use double quotes everywhere

File: example.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	console.log('Ready!');
+});
+
+client.on('messageCreate', async(msg) => {
```
Comment (MoaufmKlo) [inline example.js]: Seeing as this is async, you can use await instead of then.

**Review by MoaufmKlo — CHANGES_REQUESTED** [review]

File: example.js (hunk 3 above)
Comment (bddylol) [reply example.js]: I'll start making all changes :)

**Comment by MoaufmKlo** [comment]
@voidx999 I'd appreciate if you could take a look at my changes and suggest any necessary changes.

**Comment by MoaufmKlo** [comment]
Inactive, will close — @ me to reopen

**Pull request closed without merging**